.golangci.yml

```diff
@@ -79,7 +79,8 @@ issues:
         - gocritic
         - ineffassign
         - govet
-    - path: main.go|handlers
+        - typecheck
+    - path: main.go|handlers|cache
```

Comment:
Don't think we should be excluding typecheck for `cache` - downgrade the breaking change (which was only listed as a minor)
This linter is also a very old version